pygmt/src/surface.py

```diff
@@ -42,7 +47,22 @@ def surface(data=None, x=None, y=None, z=None, **kwargs):
     .. math::    (1 - t)\nabla^2(z)+t\nabla(z) = 0
 
     where :math:`t` is a tension factor between 0 and 1, and :math:`\nabla`
-    indicates the Laplacian operator.
+    indicates the Laplacian operator. Here, :math:`t = 0` gives the
+    “minimum curvature” solution. Minimum curvature can cause undesired
```

Comment:
Small detail, but we mostly use neutral (vertical) quotation marks (see https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html) instead of left/right quotation marks. I know this can be a bit challenging on some keyboards :slightly_smiling_face:

```suggestion
    "minimum curvature" solution. Minimum curvature can cause undesired
```